lib/format.js

```diff
@@ -0,0 +1,127 @@
+// Just enough of the minidump format to extract module names + debug
+// identifiers so we can download pdbs
```

Comment:
```suggestion
// identifiers so we can download pdbs
// https://github.com/libyal/libmdmp/blob/master/documentation/Minidump%20(MDMP)%20format.asciidoc
```

This is the reference that I used while reading this, but several of the fields there are marked 'unknown'. If you have a better reference, that would be preferred here

Reply:
I used https://chromium.googlesource.com/breakpad/breakpad/+/refs/heads/master/src/google_breakpad/common/minidump_format.h + other references in the breakpad source code.

Reply:
Sprinkled references through the code.